server/src/test/java/org/spine3/server/BoundedContextShould.java

```diff
@@ -163,14 +151,29 @@ public void register_ProjectionRepository() {
     }
 
     @Test
-    public void post_Command() {
+    public void return_unsupported_command_response_if_no_handlers_or_dispatchers() {
+        final TestResponseObserver responseObserver = new TestResponseObserver();
+        final CommandBus commandBus = boundedContext.getCommandBus();
+
+        final Command cmd = createProjectCmd();
+        boundedContext.post(cmd, responseObserver);
+
+        final Response response = responseObserver.getResponseHandled();
+        assertTrue(isUnsupportedCommand(response));
+        verify(commandBus, times(1)).store(cmd);
+        verify(commandBus, times(1)).getCommandStatusService();
```

Comment:
We need to test that real methods. “Real” are those that do what we want to test in the test. The `store()` is that kind of methods. The `getCommandStatusService()` is not. We have to call this method in the current implementation, but it is not the “meat” of the functionality under the test.